src/pages/studyView/StudyViewConfig.ts

```diff
@@ -263,6 +264,7 @@ const studyViewFrontEnd = {
             secondary: '#DC3912',
             tertiary: '#f88508',
             quaternary: '#109618',
+            mediumGrey: '#ddd',
```

Comment:
this configuration has todo with studyview.   color doesn't belong here.  see my comment about where it belongs.

Reply:
I am trying to follow the code convention/flow that was already here

Reply:
If scss importing doesn't work, what would be a good typescript location for these colors to be moved to?

Reply:
Fixed in https://github.com/cBioPortal/cbioportal-frontend/compare/765125bb1ba294e20e059cfe213bce3d7712325b..65bd1087c4c31369fa5874cd6fa05699e829a225